tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md

```diff
@@ -37,7 +37,7 @@ cd ~/projects
 3. Now that you are in the correct folder, run the following command:
 ```Shell/Bash
 mvn -B archetype:generate -DarchetypeArtifactId=cds-services-archetype -DarchetypeGroupId=com.sap.cds \
--DarchetypeVersion=1.3.0 -DcdsVersion=3.31.2 \
+-DarchetypeVersion=Release \
```

Comment:
please change `Release` to `RELEASE`

Reply:
The same change also needs to be applied to this file: https://github.com/SAPDocuments/Tutorials/blob/master/tutorials/cp-cap-java-app-studio/cp-cap-java-app-studio.md

Reply:
will do the changes in a new pr